aws/resource_aws_ami_test.go

```diff
@@ -218,30 +265,35 @@ func testAccCheckAmiEbsBlockDevice(bd *ec2.BlockDeviceMapping, ed *ec2.EbsBlockD
 	}
 }
 
-func testAccAmiConfig_basic(rName string) string {
+func testAccAmiConfig_base(size int, rName string) string {
 	return fmt.Sprintf(`
 data "aws_availability_zones" "available" {}
 
 resource "aws_ebs_volume" "foo" {
   availability_zone = "${data.aws_availability_zones.available.names[0]}"
-  size              = 8
+  size              = %d
 
   tags = {
-    Name = "testAccAmiConfig_basic"
+    Name = "%s"
   }
 }
 
 resource "aws_ebs_snapshot" "foo" {
   volume_id = "${aws_ebs_volume.foo.id}"
 
   tags = {
-    Name = "testAccAmiConfig_basic"
+    Name = "%s"
   }
 }
 
+`, size, rName, rName)
```

Comment:
Instead of repeating `fmt.Sprintf()` parameters, the format verbs can be switched to an indexed form (e.g. two `%[2]s` instead of the two `%s` verbs), then this becomes the following:

```suggestion
`, size, rName)
```

Its not such a big deal in this particular format invocation, but we have others which repeat parameters _many_ times so we would recommend this pattern as the norm instead of the exception. 😄